src/Core/Managed/Shared/Channel/InMemoryChannel.cs

```diff
@@ -10,11 +10,16 @@
     public class InMemoryChannel : ITelemetryChannel
     {
         private readonly TelemetryBuffer buffer;
-        private readonly InMemoryTransmitter transmitter;
+        private InMemoryTransmitter transmitter;
```

Comment:
Yes, originally Sergey agreed with nulling it out in Dispose but since that is gone I can make it readonly again.